### ISSUE TITLE
Use correlation as loss

### ISSUE THREAD

**Issue by bkvie** [NVlabs/PWC-Net#58]
correlation(image1, image2) outputs a multidimensional tensor, would it be possible to produce a scalar that can be used as a loss. I.e deform image 1 and minimise negative correlation between image1, image2.

asked previously and recommended to ask in this thread, [https://github.com/sniklaus/pytorch-pwc/issues/15](url)

**Comment by luvegood** [comment]
@bkvie My thinking is that you can certainly get creative and design a novel loss function, but could you please elaborate "deform image 1" and "negative correlation"?
